lib/rustler_precompiled/config.ex

```diff
@@ -71,14 +73,24 @@ defmodule RustlerPrecompiled.Config do
 
   defp validate_targets!(nil), do: raise_for_nil_field_value(:targets)
 
-  defp validate_targets!(targets) do
-    if is_list(targets) do
-      targets
-    else
-      raise "`:targets` is required to be a list of targets supported by Rust"
+  defp validate_targets!(targets) when is_list(targets) do
```

Comment:
Should we verify that at least one target is present?

```suggestion
  defp validate_targets!([_ | _] = targets) do
```